src/lib/pubkey/xmss/xmss.h

```diff
@@ -293,6 +321,8 @@ class BOTAN_PUBLIC_API(2,0) XMSS_PrivateKey final : public virtual XMSS_PublicKe
                              XMSS_Hash& hash);
 
       std::shared_ptr<XMSS_PrivateKey_Internal> m_private;
+
+      const WOTS_Derivation_Method m_wots_derivation_method;
```

Comment:
Defining this as `const` will (implicitly) remove the class' move constructor and move assignment operator. I don't think we want that.

```suggestion
      WOTS_Derivation_Method m_wots_derivation_method;
```